src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,85 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File reportFile = new File(toFileName);
+        if (!reportFile.exists()) {
+            try {
+                reportFile.createNewFile();
+            } catch (IOException e) {
+                throw new RuntimeException("Cannot create " + toFileName + " file", e);
+            }
+        }
```

Comment:
remove this, default implementation of FileWriter should do just fine for this task, no need to handle files manually